include/sound/sof.h

```diff
@@ -39,6 +39,14 @@ enum sof_fw_state {
 	SOF_FW_CRASHED,
 };
 
+/* DSP power states */
```

Comment:
garbled text in commit message Move the enum sof_dsp_power_states to include/sound/sof.h

Reply:
oops, fixed now.